interface/ceed-basis.c

```diff
@@ -699,6 +683,50 @@ int CeedHouseholderApplyQ(CeedScalar *mat_A, const CeedScalar *mat_Q, const Ceed
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Return pseudoinverse of a matrix
+
+  @param[in]     ceed      Ceed context for error handling
+  @param[in]     mat       Row-major matrix to be factorized in place
```

Comment:
Ah, no, you make a copy, so it isn't factored in place (thus destroying the original data)

Reply:
```suggestion
  @param[in]     mat       Row-major matrix to compute pseudoinverse of
```